docs/09-dev-guide.md

```diff
@@ -99,13 +92,7 @@ Finally, when you are ready to tear down the instance, run:
 ./scripts/test/tearDown.sh
 ```
 
-It is advisable to manually stop any tasks running in the compaction and system test
-clusters.
-
-Note you will still need the files in the `/generated` folder that are created
-during deployment for this tear down script to work correctly.
```

Comment:
This note still seems relevant, although you can specify the instance ID if the `/generated` files aren't there any more.